client/packages/system/src/Encounter/utils.ts

```diff
@@ -3,30 +3,32 @@ import { LocaleKey, TypedTFunction, useTranslation } from '@common/intl';
 import { EncounterNodeStatus } from '@common/types';
 import { EncounterRowFragment } from '@openmsupply-client/programs';
 
-const effectiveStatus = (
-  encounter: EncounterRowFragment,
+export const encounterStatusTranslation = (
+  status: EncounterNodeStatus,
   t: TypedTFunction<LocaleKey>
-) => {
-  const status = encounter.status;
-  if (!status) {
-    return '';
-  }
+): string => {
   switch (status) {
+    case EncounterNodeStatus.Pending:
+      return t('label.encounter-status-pending');
     case EncounterNodeStatus.Cancelled:
       return t('label.encounter-status-cancelled');
-    case EncounterNodeStatus.Completed:
-      return t('label.encounter-status-done');
-    case EncounterNodeStatus.Scheduled:
-      return t('label.encounter-status-scheduled');
-    case EncounterNodeStatus.Missed:
-      return t('label.encounter-status-missed');
+    case EncounterNodeStatus.Visited:
+      return t('label.encounter-status-visited');
     default:
-      ((_: never) => {
-        // exhaustive check
-        _;
-      })(status);
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      return ((_: never) => '')(status);
+  }
+};
+
+export const effectiveStatus = (
+  encounter: EncounterRowFragment,
+  t: TypedTFunction<LocaleKey>
+): string => {
+  const status = encounter.status;
+  if (!status) {
+    return '';
   }
-  return '';
+  return encounterStatusTranslation(status, t);
```

Comment:
Not quite sure why we need this as a separate function. Can't you just roll it into the existing `encounterStatusTranslation` and just have empty string returned as the default?

Reply:
Thanks, there was some more code in there at some point... I simplified it.